src/souffle-compile.in

```diff
@@ -107,6 +111,44 @@ cd "$(dirname $1)"
 dir="$PWD"
 cd "$OLDPWD"
 
+# Make temp folder and copy relavent files there
+if [ -n "$SWIGLANG" ]
+then
+  # set environment variables
+  # directory where swig interface files are in souffle
+  SWIGDIR=$(dirname $0)/../include/souffle/swig
+  TMP_DIR="$(mktemp -d)"
```

Comment:
Delete the temp directory before you exit. It is not cleaned up automatically.

Reply:
Fixed.